signature_verification.md

```diff
@@ -0,0 +1,19 @@
+# Signature Verification
+
+
+In endpoint-monitor, for each job we care about, we create, update and delete Nginx configurations dynamically.
+At startup time, if there is a signature-secret located at `/var/run/secrets/streams-endpoint-monitor/server-auth/signature-secret`, the function `_has_signature_secret()` in `app.py` will set the boolean value `self._signature` to true in `file_config.py`
```

Comment:
Please describe it as the customer would set a secret, not the implementation details. Ie. they set `signature. secret` in a k8s secret.

Reply:
So maybe a separate page for the external user info??

Reply:
Will do